content/en/platform/corda/5.0/application-networks/creating/notaries.md

```diff
@@ -85,25 +145,33 @@ $REGISTRATION_CONTEXT = @{
   'corda.endpoints.0.connectionURL' = "https://$P2P_GATEWAY_HOST`:$P2P_GATEWAY_PORT"
   'corda.endpoints.0.protocolVersion' = "1"
   'corda.roles.0' = "notary",
-  'corda.notary.service.name' = <An X500 name for the notary service>,
+  'corda.notary.service.name' = <notary-service-X.500-name>,
   'corda.notary.service.plugin' = "net.corda.notary.NonValidatingNotary"
```

Comment:
These fields are no longer correct as of Beta 3. We have this docs ticket that captures the required changes: https://r3-cev.atlassian.net/browse/DOC-5134

Reply:
Thanks, @relyafi. @amiegrace12 and I will look into this.
The fix version of that ticket seems to be 5.0. Should the change have been made for the Beta 3 release?

Reply:
Hi @nadinequinr3, yes I think you're right, the original ticket (+ docs) was originally slated for GA but was brought forward to Beta 3 as it was ready in time. I'll change these now, sorry about that.